p2p/host/relay/autorelay_test.go

```diff
@@ -144,7 +145,9 @@ func connect(t *testing.T, a, b host.Host) {
 
 // and the actual test!
 func TestAutoRelay(t *testing.T) {
-	//t.Skip("fails 99% of the time")
+	if cienv.IsRunning() {
+		t.Skip("disabled on CI: fails 99% of the time")
```

Comment:
well it's not 99% of the time, it's more like 50%.... but it's ok, we can exaggerate a bit :)

Reply:
(fair enough)